script/single-deployment/CowAmmModule.s.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import {console} from "forge-std/Script.sol";
+
+import {CowAmmModule} from "src/CowAmmModule.sol";
+
+import {Utils} from "script/libraries/Utils.sol";
+
+contract DeployCowAmmModule is Utils {
+    address internal handler;
+
+    constructor() {
+        solutionSettler = addressEnvOrDefault("SETTLEMENT_CONTRACT", DEFAULT_SETTLEMENT_CONTRACT);
+        extensibleFallbackHandler =
+            addressEnvOrDefault("EXTENSIBLE_FALLBACK_HANDLER_CONTRACT", DEFAULT_EXTENSIBLE_FALLBACK_HANDLER_CONTRACT);
+        composableCow = addressEnvOrDefault("COMPOSABLE_COW_CONTRACT", DEFAULT_COMPOSABLE_COW_CONTRACT);
+        handler = addressEnvOrDefault("CONSTANT_PRODUCT_CONTRACT", address(0));
+
+        console.log("Settlement contract at %s.", solutionSettler);
+        console.log("ExtensibleFallbackHandler contract at %s.", extensibleFallbackHandler);
+        console.log("ComposableCoW contract at %s.", composableCow);
+
+        assertHasCode(solutionSettler, "no code at expected solution settler contract");
+        assertHasCode(extensibleFallbackHandler, "no code at expected extensible fallback handler contract");
+        assertHasCode(composableCow, "no code at expected composable cow contract");
+
+        // Special case: if the constant product contract is not provided, we assume
+        // that this script is being called from DeployAllContracts and that the
+        // constant product contract will be deployed by it.
+        if (handler == address(0)) {
+            console.log("No constant product contract provided, assuming called in deployAll.");
+        } else {
```

Comment:
My weak preference is not printing anything if `handler` is zero, it's weird to read that every time you deploy all contracts.